executor/agents/agno/agno_agent.py

```
@@ -210,6 +210,73 @@ def update_prompt(self, new_prompt: str) -> None:
+                        self.single_agent = self.team.agents[0]
+
+                    logger.info(f"Successfully updated model configuration to: {new_model_id}")
+
+                    # Clear existing client to force recreation on next execution
+                    if self.session_id in self._clients:
+                        try:
+                            if hasattr(self._clients[self.session_id], 'close'):
+                                await self._clients[self.session_id].close()
+                            del self._clients[self.session_id]
+                            logger.info(f"Closed existing client for session_id: {self.session_id}")
+                        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)